tests/integration/block_createBlock_api_test.go

```diff
@@ -37,17 +37,22 @@ var (
 )
 
 func TestBlockImageCreateSuite(t *testing.T) {
-	suite.Run(t, new(BlockImageCreateSuite))
+	s := new(BlockImageCreateSuite)
+	defer func(s *BlockImageCreateSuite) {
+		HandlePanics(recover(), s.o, s.T)
```

Comment:
yes, i like how every suite just needs to call the same single function now, and it actually simplified the `TearDownSuite()` functions too.

Reply:
i am going to the same to setUp as well, I didnt do it in this PR becuase i had another PR where i created new tests, that is master now, and i can updated all tests as once.